Packs/FireEyeNX/Integrations/FireEyeNX/FireEyeNX.py

```diff
@@ -224,6 +224,7 @@ def is_supported_context_type(content_type: str):
             content_type == 'application/pdf'
```

Comment:
for bonus elegance points, replace `if (... or ... ): return True` with `return (... or ...)`